index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="ku">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="her yêke û şitllêk biçênîn, Daristanêk pêk dênîn">
```

Comment:
```suggestion
    <meta name="description" content="Her yêke û şitllêk deçênîn, Daristanêk pêk dênîn">
```
Tikeye diruşimekey çak, be hele nûsirawe